core/arch/arm/plat-synquacer/main.c

```diff
@@ -16,7 +16,7 @@
 #include <kernel/timer.h>
 #include <mm/core_memprot.h>
 #include <platform_config.h>
-#include <rng_pta.h>
+#include <synq_rng_pta.h>
```

Comment:
I'd rather use double quotes to make it clear it's a local file (and keep the current name). If the name is change then `synquacer_rng_pta.h` seems preferable to `synq_rng_pta.h`.

Reply:
double :+1: